src/shared/components/common/star-rating/index.tsx

```diff
@@ -0,0 +1,98 @@
+'use client';
+
+import { useState, useEffect } from 'react';
+
+type StarRatingProps = {
+  value?: number;
```

Comment:
사용자가 직접 별점을 메기는 UX이기 때문에 `value` 값은 기본 값을 가지면 안 되고, 반드시 외부에서 전달되어야 합니다.
```suggestion
  value: number;
```